applications/jonny5/src/j5_limits.erl

```diff
@@ -394,4 +403,5 @@ create_limits(AccountId, AccountDb, JObj) ->
            ,allotments = kz_json:get_value(<<"pvt_allotments">>, JObj, kz_json:new())
            ,soft_limit_inbound = get_limit_boolean(<<"soft_limit_inbound">>, JObj, 'false')
            ,soft_limit_outbound = get_limit_boolean(<<"soft_limit_outbound">>, JObj, 'false')
+           ,inbound_channels_per_did = kz_json:get_value(<<"pvt_inbound_channels_per_did">>, JObj, kz_json:new())
```

Comment:
This base branch for this is out of date, all these values were moved to kzd_limits. Can you please rebase and then move this to the kzd_limits. This should follow the other limits parameters, which define a default but seek up to the system_config.limits if nothing is defined on the account limits doc in public or private fields.

Reply:
@swysor thanks for looking at this pr. will rebase it next week, regards,

Reply:
Sounds good, sorry for the delay in getting to this! I will keep an eye for updates on this one.